test/specs.test.js

```diff
@@ -186,16 +191,17 @@ lab.experiment('api', () => {
       expect(context).to.equal({ yaml: {},
         method: {},
         inject: {},
-        api: [{ userId: 1,
-          id: 1,
-          title: 'sunt aut facere repellat provident occaecati excepturi optio reprehenderit',
-          body: 'quia et suscipit\nsuscipit recusandae consequuntur expedita et cum\nreprehenderit molestiae ut ut quas totam\nnostrum rerum est autem sunt rem eveniet architecto' }
-        ]
+        api: {
+          _: [{ userId: 1,
```

Comment:
we shouldn't be returning the _, if it's `_` then it should set it on the top level.  this will break all of our current code